lib/classes/CLI.js

```diff
@@ -231,9 +233,17 @@ class CLI {
       )
     );
 
+    this.consoleLog('');
+    this.consoleLog(chalk.yellow.underline('General Commands'));
     this.consoleLog('');
     if (Object.keys(this.loadedCommands).length) {
-      Object.entries(this.loadedCommands).forEach(([command, details]) => {
+      const internalCommands = Object.entries(this.loadedCommands).filter(
+        command => command && !command.isExternal
```

Comment:
I think this doesn't filter as expected, as in `Object.entries`  result, each item is a `[key, value]` array, so `command` here is an array.

Simple fix would be to use `Object.values` instead, which is also more appriopriate in this case